frontend/FarmWise/app/src/main/java/com/example/farmwise/FarmFragment.java

```diff
@@ -1,11 +1,81 @@
+//package com.example.farmwise;
```

Comment:
this file can probably be removed if everything is commented out

Reply:
Working on fixing it

Reply:
Removed files from build and changed the commented file, the rest of the files are part of the language model I used and the grammar I created for crops so they are needed.